pac4j-core/src/main/java/org/pac4j/core/io/Resource.java

```diff
@@ -1,21 +1,21 @@
 package org.pac4j.core.io;
 
+import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 
 /**
  * Interface for a resource descriptor that abstracts from the actual
  * type of underlying resource, such as a file or class path resource.
- *
+ * <p>

```

Comment:
Looks like Travis is skipping Javadocs: `mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V`